### PR TITLE
fix: widget ui bugs

### PR DESCRIPTION
**Pull request by mikasackermn**
# Summary

fixed bugs in widget on small screens

Fixes # (issue)
- [x] fix blockchain alignment in small screens
- [x] not found message is not aligned correctly (check different pages)
- [x] warning message text isn't correct
- [x] notification drop down in small screens
- [x] Wallet addresses on small screens are larger than the size of the div
- [x] fix blockchains section show in small screens

# Checklist:

- [x] I have performed a self-review of my code
- [x] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

### CODE REVIEW

File: widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts [hunk 1]
```diff
@@ -3,5 +3,8 @@ import { styled } from '@rango-dev/ui';
 export const Blockchains = styled('div', {
   display: 'grid',
   gap: '$10',
-  gridTemplateColumns: 'repeat(6, minmax(0, 1fr))',
+  gridTemplateColumns: 'repeat(4, minmax(0, 1fr))',
```
Comment (RyukTheCoder) [inline widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts:6]: Now blockchain chips are not equal in size. You can solve this problem by setting `width: '100%'` for BlockchainChip.

image

**Review by RyukTheCoder — CHANGES_REQUESTED** [review]
Well done. I've reviewed your changes and have just added a single comment for your consideration.

File: widget/embedded/src/components/NotificationContent/NotificationContent.styles.ts [hunk 2]
```diff
@@ -2,8 +2,11 @@ import { styled } from '@rango-dev/ui';
 
 export const Container = styled('div', {
```
[the remark this answers is not in the document]
Comment (mikasackermn) [reply widget/embedded/src/components/NotificationContent/NotificationContent.styles.ts:3]: fixed

File: widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx [hunk 3]
```diff
@@ -141,7 +141,7 @@ export function WalletList(props: PropTypes) {
           walletType: wallet.type,
           chain,
         });
-        const conciseAddress = address ? getConciseAddress(address) : '';
+        const conciseAddress = address ? getConciseAddress(address, 7) : '';
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx:144]: I believe the 'no-magic-numbers' ESLint rule has been deactivated in this file during previous modifications. Could you please eliminate it from the beginning of the file and introduce a constant for the number (7)?

File: widget/embedded/src/components/NotificationContent/NotificationContent.styles.ts [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { styled } from '@rango-dev/ui';
...
   padding: '$10',
   width: '350px',
+  maxWidth: '90vw',
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/components/NotificationContent/NotificationContent.styles.ts:6]: Can you modify the popover to exclusively display within the boundaries of the widget?

File: widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts [hunk 5]
```diff
@@ -3,5 +3,8 @@ import { styled } from '@rango-dev/ui';
 export const Blockchains = styled('div', {
   display: 'grid',
   gap: '$10',
-  gridTemplateColumns: 'repeat(6, minmax(0, 1fr))',
+  gridTemplateColumns: 'repeat(4, 1fr)',
+  '@xs': {
+    gridTemplateColumns: 'repeat(6, 1fr)',
+  },
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts:9]: Perhaps I am mistaken, but upon reviewing this component with the previous styles, the alignment and size of the blockchain section appeared to be fine. However, with the recent modifications, the blockchain section now seems longer, and the token list appears shorter. Could you please verify if the output matches the image when using the previous styles? If it does, consider reverting your recent changes.

File: widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx (hunk 3 above)
Comment (mikasackermn) [reply widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx]: fixed

File: widget/embedded/src/components/NotificationContent/NotificationContent.styles.ts (hunk 4 above)
Comment (mikasackermn) [reply widget/embedded/src/components/NotificationContent/NotificationContent.styles.ts:6]: fixed

File: widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts [hunk 6]
```diff
@@ -3,5 +3,5 @@ import { styled } from '@rango-dev/ui';
 export const Blockchains = styled('div', {
   display: 'grid',
   gap: '$10',
-  gridTemplateColumns: 'repeat(6, minmax(0, 1fr))',
+  gridTemplateColumns: 'repeat(6, 1fr)',
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts:6]: Considering the comments made earlier, could you please reverse these alterations?

Comment (mikasackermn) [reply widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts]: fixed

File: widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx [hunk 7]
```diff
@@ -42,6 +41,7 @@ import {
   WalletImageContainer,
 } from './WalletList.styles';
 
+const EllipsisLength = 7;
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx:44]: Can you ensure that the constants follow our naming convention? Additionally, I believe there might be an error in the name.
ACCOUNT_ADDRESS_MAX_CHARACTERS

Comment (mikasackermn) [reply widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx]: fixed

**Review by Ikari-Shinji-re — APPROVED** [review]

**Pull request merged**